### PR TITLE
Fabulous graphics toggle - Sodium edition

### PR DESCRIPTION
**Pull request by IMS212**
A bit hacky, but seems to work well.

### CODE REVIEW

**Review by coderbot16 — CHANGES_REQUESTED** [review]
There's a lot of code duplicated between these code paths - deduplicating them would be preferable.

**Comment by IMS212** [comment]
The code here is quite messy, however it's been deduplicated.

File: src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptionPages.java [hunk 1]
```diff
@@ -155,20 +157,25 @@ GraphicsMode toVanilla() {
 
     public static OptionPage quality() {
         List<OptionGroup> groups = new ArrayList<>();
+        OptionImpl.Builder graphicsQuality;
+        if(!Iris.getIrisConfig().areShadersEnabled() && GlStateManager.supportsGl30()) {
```
Comment (coderbot16) [inline src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptionPages.java]:
```suggestion
        if (!Iris.getIrisConfig().areShadersEnabled() && GlStateManager.supportsGl30()) {
```

**Review by coderbot16 — APPROVED** [review]
One whitespace change, otherwise looks good to me.

**Pull request merged**